pkgs/tools/archivers/arc_unpacker/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake, writeText, boost, libpng, libjpeg, zlib
+, openssl, libwebp, catch }:
+
+stdenv.mkDerivation rec {
+  pname = "arc_unpacker";
+  version = "b9843a13e2b67a618020fc12918aa8d7697ddfd5";
```

The version should not just be the git rev, instead use this `unstable-${commit-date}` format.
```suggestion
  version = "unstable-2019-01-28";
```

---

Fixed